package.json

```diff
@@ -46,16 +46,19 @@
   },
   "dependencies": {
     "@kadira/react-split-pane": "^1.4.0",
-    "redux": "^3.5.2",
-    "mantra-core": "^1.6.1",
     "babel-runtime": "^6.5.0",
+    "classnames": "^2.2.5",
```

Comment:
This one and the fuse.js is not directly used in this repo. Why don't you put this directly inside the `react-fuzzy` package.